Makefile

```diff
@@ -7,7 +7,6 @@ TEST_OS = fedora-40
 endif
 export TEST_OS
 TARFILE=$(RPM_NAME)-$(VERSION).tar.xz
-NODE_CACHE=$(RPM_NAME)-node-$(VERSION).tar.xz
```

Comment:
What happened to this? We won't be able to get this into Fedora without rebuilding the bundle during rpm build.